index.html

```diff
@@ -4,191 +4,205 @@
+                <h2 class="mobile-h2">Profesional Art <br> Printing Data</h2>
+                <h2 class="desktop-h2">Data Dashboard <br> Healthcare</h2>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                <div class="stacktwo">
-                    <h3>css</h3>
-                    <h3>bootstrap</h3>
-                    <h3>Ruby</h3>
-                </div>
-                <button><a href="#">See Project</a></button>
+                <ul class="stacktwo">
+                    <li>css</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button" class="worktwo-button"><a href="#">See Project</a></button>
             </div>
-            
 
-            <!-- work two sixth container -->
-            <div class="worktwo">
-                <h2>Profesional Art <br> Printing Data</h2>
+            <div class="worktwo box36">
+                <h2 class="mobile-h2">Profesional Art <br> Printing Data</h2>
+                <h2 class="desktop-h2">Website Protfolio</h2>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                <div class="stacktwo">
-                    <h3>css</h3>
-                    <h3>bootstrap</h3>
-                    <h3>Ruby</h3>
-                </div>
-                <button><a href="#">See Project</a></button>
+                <ul class="stacktwo">
+                    <li>css</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button" class="worktwo-button"><a href="#">See Project</a></button>
             </div>
         </div>
     </section>
 
+    
+    
     <!-- About Section -->
-    <section class="about-container"> 
+    <section class="aboutme-container"> 
         <div class="aboutme">
-            <h1>About me</h1>
-            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <a href="#">Get My Resume</a>
-        </div>
-        <div class="picture-background">
-            <img src="./images/illustration-about-me-mobile.png" alt="">
+            <div>
+                <h1>About me</h1>
+                <p class="aboutme-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a href="#">Get My Resume</a>
+            </div>
+            <img class="aboutme-img" src="./images/about-img-desktop.png" alt="">
         </div>
         <div class="end-line"></div>
```

Comment:
- **_[OPTIONAL]_**  You have an extra background image in your mobile design 😵, and this background image is for the desktop screens, so when the screen is shrunk to smaller sizes it's still there 👇🏽.

### Figma design
![image](https://user-images.githubusercontent.com/43172164/206839819-20dee12e-9676-404c-a8eb-8a2bf691a97b.png)


### Your mobile design
![image](https://user-images.githubusercontent.com/43172164/206839801-f6a9ff55-887c-458c-b668-962b23995532.png)